luma/core/device.py

```diff
@@ -255,8 +267,20 @@ def display(self, image):
         assert image.size == self.size
 
         image = self.preprocess(image)
-        path = f"/dev/fb{self.id}"
-        data = bytes(self.__image_converter(image))
-
-        with open(path, "wb") as fp:
-            fp.write(data)
+        file_handle = self.__file_handle
```

Comment:
is this ref necessary?

Reply:
primarily: cheaper to reference local variable than repeatedly lookup of self.variable in loop a bit later on in the code, but TBH probably a micro-optimization